src/accountEnvironmentsList.ts

```diff
@@ -0,0 +1,7 @@
+import { Environment } from "@azure/ms-rest-azure-env";
+import { AzureAccountEnvironment } from "./azure-account.api";
+
+export const accountAzureCloud: AzureAccountEnvironment = {...Environment.AzureCloud, azureStackApiProfile: false};
```

Comment:
I believe it is just an addition of extra property, therefore it is not breaking change for the existing consumers of this API

Reply:
Make api profile an optional property